Sources/ScipioKit/Producer/PIF/ToolchainGenerator.swift

```diff
@@ -60,3 +60,36 @@ struct ToolchainGenerator {
         )
     }
 }
+
+fileprivate extension ToolchainGenerator {
+
+    /// A non-caching environment-aware implementation of `SwiftSDK.sdkPlatformFrameworkPaths`
+    /// Returns `macosx` sdk platform framework path.
+    func resolveSDKPlatformFrameworkPaths() async throws -> (frameworkPath: AbsolutePath, libPath: AbsolutePath) {
```

Comment:
```suggestion
extension ToolchainGenerator {

    /// A non-caching environment-aware implementation of `SwiftSDK.sdkPlatformFrameworkPaths`
    /// This implementation is based on the original SwiftPM
    /// https://github.com/swiftlang/swift-package-manager/blob/release/6.0/Sources/PackageModel/SwiftSDKs/SwiftSDK.swift#L592-L595
    /// Returns `macosx` sdk platform framework path.
    fileprivate func resolveSDKPlatformFrameworkPaths() async throws -> (frameworkPath: AbsolutePath, libPath: AbsolutePath) {
```

Reply:
Thanks, referring to a particular LoC is better, indeed.